src/intersection-observer.js

```diff
@@ -219,6 +219,10 @@ export class IntersectionObserver extends Observable {
    * @private
    */
   flush_() {
+    if (!this.iframe_) {
+      return;
```

Comment:
If `!this.iframe` will this `IntersectionObserver` be reused. Do we need to do some clean up here?
We can probably set `this.boundFlush = null`, `this.pendingChanges_ = null` here?